src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue

```diff
@@ -1322,10 +1316,6 @@ export default {
 			this.$nextTick(() => {
 				// re-enable visibility of loans, activates loan fetch within loan display
 				this.showLoans = true;
-				// up loan query for grid view
-				if (!this.showLoanRows && this.$refs.loandisplayref && this.$route.query.page) {
-					this.$refs.loandisplayref.updateFromParams(this.$route.query);
```

Comment:
This might break paging in grid view...

Reply:
In what way?  I tried switching pages in grid view and to me it appears to work just fine...

Reply:
If you refresh the page, the grid view doesn't load and the `x` page of loans aren't loaded...

Reply:
I do see that paging generally works though in grid view!

Reply:
I'm having trouble reproducing this issue...  Here's a video of me testing and it appearing to work.  Maybe you could let me know if there's a discrepancy between what I did and what you did?
https://www.loom.com/share/eeb16160ab5b430c8617a74cb1fc1108